src/Nexus.ts

```diff
@@ -16,6 +18,7 @@ type NexusConfig = {
 // Without this, jest crashes really bad...
 if (process.env.NODE === 'TEST') {
   Organization;
+  Realm;
```

Comment:
Too bad we have to do this for Jest 😢

Reply:
yeah I dont understand what's going on. I think in the future we should just dump jest for a more barebones testing framework?